### DISCUSSION TITLE
Visualizing stress and strain

### DISCUSSION THREAD

**Discussion by janetunde** [JWock82/Pynite#136] [Q&A] [answered]
Hi all!
Quick question from a beginner, thank you for your patience! Is there any analog to visualize local stress and strain in the regions along a member's length as we do with meshes? Or no, because we only define cross-sectional area of the members?
Thank you very much.

**Comment by JWock82** [comment, marked as answer]
No. The closest thing is that you can plot the moment, shear, and axial force diagrams for each member about each major axis.

That said, the stress components at any point in the cross-section can be calculated using classical equations:
- Bending stress = M/S
- Shear stress = VQ/I
- Axial stress = P/A

**Reply by janetunde** [reply]
Many thanks!! :)
